### PR TITLE
Implement consistent term for installation related flows

### PR DESCRIPTION
**Pull request by rcaballeromx**

### CODE REVIEW

File: content/docs/setup/kubernetes/additional-setup/cni/index.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 title: Install Istio with the Istio CNI plugin
-description: Instructions to install and use Istio with the Istio CNI plugin, allowing operators to deploy services with lower privilege.
+description: Install and use Istio with the Istio CNI plugin, allowing operators to deploy services with lower privilege.
```
Comment (venilnoronha) [inline content/docs/setup/kubernetes/additional-setup/cni/index.md]: *CNI* and *Container Network Interface* terms are used interchangeably. I'd prefer if we specify *CNI (Container Network Interface)* at the first time and use *CNI* in the rest of the document.

File: content/docs/setup/kubernetes/install/_index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Installation Paths
-description: Choose the path that best suits your needs and platform.
+title: Installation Flows
+description: Choose the flows that best suits your needs and platform.
```
Comment (venilnoronha) [inline content/docs/setup/kubernetes/install/_index.md]: nit: s/suits/suit

File: content/docs/setup/kubernetes/additional-setup/cni/index.md (hunk 1 above)
Comment (rcaballeromx) [reply content/docs/setup/kubernetes/additional-setup/cni/index.md]: @venilnoronha 
I agree but that is outside the scope of the PR and we can fix it post 1.1

**Comment by rcaballeromx** [comment]
@venilnoronha Addressed your comments and fixed lint, PTAL

**Pull request merged**